modules/generate_temp_lightsail_access/main.py

```diff
@@ -126,21 +126,21 @@ def main(args, pacu_main):
                     protocol=protocol
                 )
                 temp_keys[region].append(response['accessDetails'])
-                print('  Successfully created temporary access for {}'.format(name))
+                print('    Successfully created temporary access for {}'.format(name))
             except ClientError as error:
                 code = error.response['Error']['Code']
                 if code == 'AccessDeniedException':
-                    print('Unauthorized to generate temporary access.')
+                    print('     Unauthorized to generate temporary access.')
                     return
                 elif code == 'OperationFailureException':
-                    print(error.response['Error']['Message'])
+                    print(error.response['Error']['Code'])
```

Comment:
Is this just printing 'OperationFailureException' everytime? Why is it caught explicitly?

Reply:
OperationFailureException generally occurs when you try to generate temporary access for a non running instance. I can be more explicity in the failure message.